.github/workflows/test-and-release.yaml

```diff
@@ -39,10 +39,22 @@ jobs:
           PUBLIC_VERSION: ${{ github.sha }}
         run: npm run build
 
+      - name: Install playwright
+        run: npx playwright install
+
+      - name: Start server
+        run: npx node-static build > server.log 2>&1 &
+
+      - name: Wait for server to be up
+        run: until nc -w 10 127.0.0.1 8080; do sleep 1; done
```

Comment:
that might fit for your use-case: https://playwright.dev/docs/test-advanced#launching-a-development-web-server-during-the-tests

Reply:
Yeah, I saw that, but in the current way I can have the dev server running while developing and use the PW debugger.